draft-ietf-acme-acme.md

```diff
@@ -1086,7 +1086,7 @@ Content-Type: application/problem+json
 Content-Language: en
 
 {
-  "type": "urn:ietf:params:acme:error:agreementRequired"
+  "type": "urn:ietf:params:acme:error:userActionRequired"
   "detail": "Terms of service have changed"
```

Comment:
You can imagine other cases where you might want to break automation.  For example, if a user account has been dormant for a long time, and you want to do some advanced verification.  I think we can leave it up to CAs to determine in what cases they want to use this error.

However, according to [RFC 7807](https://tools.ietf.org/html/rfc7807#section-3.1), the `detail` field contains `human-readable explanation specific to this occurrence of the problem`.  So even if there are other use cases, it makes sense for this field to be specific to this use case.